packages/textlint-tester/test/textlint-tester-invalid-testcofig-test.js

```diff
@@ -0,0 +1,150 @@
+        config: {
+            rules: [],
+            plugins: [
+                {
+                    pluginId: "html",
+                    plugin: htmlPlugin
+                }
+            ]
+        },
+        expectedErrorMessage: "TestConfig.rules should have at least one rule"
+    },
+    {
+        description: "One of rules has no ruleId",
+        config: {
+            rules: [
+                {
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "ruleId property not found"
+    },
+    {
+        description: "One of rules has no rule",
+        config: {
+            rules: [
+                {
+                    ruleId: "no-todo"
+                }
+            ]
+        },
+        expectedErrorMessage: "rule property not found"
+    },
+    {
+        description: "TestConfig has plugins proeprty which is not an array",
+        config: {
+            plugins: htmlPlugin,
+            rules: [
+                {
+                    ruleId: "no-todo",
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "TestConfig.plugins should be an array"
+    },
+    {
+        description: "One of plugins has no pluginId",
+        config: {
+            plugins: [
+                {
+                    plugin: htmlPlugin
+                }
+            ],
+            rules: [
+                {
+                    ruleId: "no-todo",
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "pluginId property not found"
+    },
+    {
+        description: "One of plugins has no plugin",
+        config: {
+            plugins: [
+                {
+                    pluginId: "html"
+                }
+            ],
+            rules: [
+                {
+                    ruleId: "no-todo",
+                    rule: noTodoRule
+                }
+            ]
+        },
+        expectedErrorMessage: "plugin property not found"
+    }
+];
+
+describe("new-style-of-test: invalid testConfig", () => {
+    testConfigs.forEach(testConfig => {
+        it(`Should throw assertion error: ${testConfig.description}`, () => {
+            try {
+                tester.run("invalid-testConfig-test", testConfig.config, {
+                    valid: [
+                        {
+                            text: "text",
+                            ext: ".md"
+                        }
+                    ],
+                    invalid: [
+                        {
+                            text: "- [ ] string",
+                            ext: ".md",
+                            errors: [{ message: "Found TODO: '- [ ] string'", line: 1, column: 3 }]
+                        }
+                    ]
+                });
+            } catch (err) {
+                assert.equal(err.code, "ERR_ASSERTION");
```

Comment:
Does `err instanceof assert.AssertError` work?

Reply:
Works as expected on Node.js v{4,5,6,7,8,9} on my end. Thank you!